server/src/stateManagement.mjs

```diff
@@ -160,6 +137,7 @@ function getUserId(userId){
   // //Checking if user or appId is present in state object
   if (!(userId in state)){
     for(var key in users){
+
```

Comment:
I still see uses of "~", which seems less than ideal. Once the currentUserId is parsed and each userId is parsed within the loop, I don't think there should be any string-related manipulation going on.

Reply:
Thanks Mike for pointing it out,We have separate ticket to handle the user inside the loop.This ticket is only for ParsedUserId().Here is the ticket:https://ccp.sys.comcast.net/browse/SYSTEST-9362